### PR TITLE
Action bar captions

### PR DESCRIPTION
**Pull request by JPLachance**
* Update table-actions example
* Get rid of the display table in coveo-table-actions
* Align all icons vertically

### CODE REVIEW

File: docs/_components/tables/03_table-actions.html [hunk 1]
```diff
@@ -7,11 +7,106 @@
...
+                                <ellipse ry=".7" rx=".7" cy="10" cx="13.6"></ellipse>
+                                <path d="M13.6 11.2c-.7 0-1.2-.5-1.2-1.2s.5-1.2 1.2-1.2 1.2.5 1.2 1.2-.5 1.2-1.2 1.2zm0-1.4c-.1 0-.2.1-.2.2 0 .2.4.2.4 0 0-.1-.1-.2-.2-.2z"></path>
+                            </g>
+                            <path d="M10 17.2c-4 0-7.2-3.2-7.2-7.2S6 2.8 10 2.8 17.2 6 17.2 10 14 17.2 10 17.2zM10 1c-5 0-9 4-9 9s4 9 9 9 9-4 9-9-4-9-9-9z"></path>
+                        </svg>
+                    </span>
+                </span>
+                <ul class="dropdown-menu">
+                    <li>
+                        <span class="enabled" data-trigger="indexes:edit-json" data-icon="" data-requires-confirmation="" data-unrepeatable="">Action</span>
```
Comment (marilynechan) [inline docs/_components/tables/03_table-actions.html]: I'd make the labels, data-trigger, etc. more generic.

File: scss/tables/table-actions.scss [hunk 2]
```diff
@@ -5,17 +5,23 @@
...
   .coveo-table-actions {
-    position: relative;
-
-    display: table;
-    height: $table-actions-container-height;
     padding: 0 10px;
 
-    .action {
-      display: table-cell;
+    .actions-row {
```
Comment (marilynechan) [inline scss/tables/table-actions.scss]: Personnally, I liked action-bar better. Opened to discussion!

**Comment by marilynechan** [comment]
Nothing more to say, :ship: it!

**Comment by pochretien** [comment]
LGTM! :shipit:

File: docs/_components/tables/03_table-actions.html (hunk 1 above)
Comment (JPLachance) [inline docs/_components/tables/03_table-actions.html]: I removed all data tags, labels are the generics ones fitting with icons.

**Pull request merged**